superset-frontend/plugins/plugin-chart-pivot-table/src/plugin/controlPanel.tsx

```diff
@@ -90,15 +89,40 @@ const config: ControlPanelConfig = {
         ],
         ['adhoc_filters'],
         emitFilterControl,
+        ['limit'],
         [
           {
             name: 'row_limit',
             config: {
               ...sharedControls.row_limit,
+              label: t('Cell limit'),
+              description: t('Limits the number of cells that get retrieved.'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'timeseries_limit_metric',
```

Comment:
Let's rather use `series_limit_metric`, as `timeseries_limit_metric` is now deprecated.

Reply:
👍